lib/cache-expiry.js

```diff
@@ -4,15 +4,28 @@ module.exports = function getCacheExpiryGenerator (opts) {
 
   /**
    * Get the expiry for a response. It might be just the default, but we also
-   * support custom expiry length for certain status codes
+   * support custom expiry length for certain status codes.
+   *
+   * By default a non 200 status is not cached (return 0)
+   *
    * @param  {String} statusCode
    * @return {Number}
    */
   return function getCacheExpiry (statusCode) {
-    return (
-      opts.statusCodeExpires && opts.statusCodeExpires[statusCode] ||
-      opts.expeditious.getDefaultTtl()
-    );
+    if (
+      statusCode !== 200 &&
```

Comment:
This logic should be flipped around (then the block switched with the else block):

```js
if (statusCode === 200 && ...)
```